x/ecocredit/client/query.go

```diff
@@ -495,3 +496,41 @@ $%s q %s projects-by-reference-id R1 --limit 10
 
 	return qflags(cmd)
 }
+
+// QueryProjectsByAdminCmd returns command that retrieves list of projects by admin with pagination.
+func QueryProjectsByAdminCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "projects-by-admin [admin]",
+		Short: "Retrieve list of projects by admin with pagination flags",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Retrieve list of projects by admin with pagination flags
+			
+Examples:
+$%s query %s projects-by-admin regenx1v44...
+$%s q %s projects-by-admin regenx1v44.. --limit 10
```

Comment:
We should use `Example` for examples.

Documentation generated using `Example`:

- https://docs.regen.network/commands/regen_query_ecocredit_basket.html

Documentation with examples in the `Long` description:

- https://docs.regen.network/commands/regen_query_ecocredit_basket-balance.html